charts/openyurt/templates/yurt-manager-auto-generated.yaml

```diff
@@ -6,985 +6,206 @@
 # ---------------------------------------------------
```

Comment:
this file is different from the file that generated by `make manifests` command.